### ISSUE TITLE
may i know what this project is about??

### ISSUE THREAD

**Issue by jfkdcman** [kryzodoze/Delaygram#1]
it seems intresting//   may i know what this project is about??

**Comment by keiththomas1** [comment]
Yeah it was basically aiming to be a parody game of Instagram. So it had some basic Instagram features like ability to post photos and then other users could see your post in their feed. And there is a Tinder-like system for rating images which gave points to you if people liked your photo. The parody part was going to lean into the concept of people wanting to be influencers and giving up their entire physical lives to live this fantasy social media life.

**Comment by jfkdcman** [comment]
wow! is all that real posting, or the posting is all a game??

**Comment by keiththomas1** [comment]
Real posting. Servers aren't up anymore, but you can see the entire server in the 'server' folder. Used Mongo DB as storage, and just provided a simple API for getting/posting "social media posts".

**Comment by jfkdcman** [comment]
sir, im a student and dont know server mongo DB...  can you help me setup
the app if i give you 10$ from my savings??

I know its not much, but i really want to learn during this vacation

**Comment by keiththomas1** [comment]
Sorry can't help you there! It's been a few years since I've set this up so I would have to google it myself. If it helps, I just used the free service at https://www.mongodb.com/atlas/database and followed some guide and it was pretty straight-forward.
